src/kademlia/dht_storage.cpp

```diff
@@ -453,6 +464,15 @@ namespace
 			touch_item(i->second, addr);
 		}
 
+		int get_infohashes_sample(entry& item) const override
+		{
+			item["interval"] = clamp(m_settings.sample_infohashes_interval, 0, 21600);
```

Comment:
this literal integer, I take it, is defined by the bep? what do you think about giving it a name (as a constant) and use that instead?